src/style.css

```
@@ -0,0 +1,49 @@
+body {
+  margin: 0;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars--1 .stars__star:nth-child(1) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--2 .stars__star:nth-child(1),
+.stars--2 .stars__star:nth-child(2){
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--3 .stars__star:nth-child(1),
+.stars--3 .stars__star:nth-child(2),
+.stars--3 .stars__star:nth-child(3){
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--4 .stars__star:nth-child(1),
+.stars--4 .stars__star:nth-child(2),
+.stars--4 .stars__star:nth-child(3),
+.stars--4 .stars__star:nth-child(4) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--5 .stars__star:nth-child(1),
+.stars--5 .stars__star:nth-child(2),
+.stars--5 .stars__star:nth-child(3),
+.stars--5 .stars__star:nth-child(4),
+.stars--5 .stars__star:nth-child(5N) {
```

you can use a simpler selector. please fix it for all classes and merge similar styles
```suggestion
.stars--5 .stars__star:nth-child(-n+5) {
```